Automattic/jetpack
# Admin: only show license notice to main connection owner

#### Changes proposed in this Pull Request:

**I'm not sure I took the right approach with this PR**.

Currently, we can get into situations where we cannot properly display the user license activation notice:

<img width="1392" alt="image" src="https://user-images.githubusercontent.com/426388/183039215-fc024693-c2ec-4a42-9b20-82c240859a7a.png">

This notably happens when you're logged in to a site with an editor account, that is connected to WordPress.com.

I think only the connection owner should be able to add a license to a site, so we shouldn't have to display a license activation notice to editors on the site.

#### Other information:


#### Jetpack product discussion

* N/A

#### Does this pull request change what data or activity we track or use?

* No

#### Testing instructions:

* Start with a site that you've connected to WordPress.com.
* Add a new Editor user.
* Log in with that editor user, and connect that account to a new WordPress.com account.
* Go to https://cloud.jetpack.com/pricing and purchase a product, like Backup. **Do not add it to any site**
* Go back to the site's admin, and try to load the Jetpack dashboard page.
    * It should load without any issues.
